src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -2532,6 +2532,389 @@ void spiral(index_t ndoms,
     }
 }
 
+//---------------------------------------------------------------------------//
+void venn_full_matset(Node &res,
+                      index_t nx,
+                      index_t ny)
+{
+    // create the material sets
+
+    index_t elements = nx * ny;
+
+    res["matsets/matset/topology"] = "topo";
+    res["matsets/matset/volume_fractions/background"] = DataType::float64(elements);
+    res["matsets/matset/volume_fractions/circle_a"] = DataType::float64(elements);
+    res["matsets/matset/volume_fractions/circle_b"] = DataType::float64(elements);
+    res["matsets/matset/volume_fractions/circle_c"] = DataType::float64(elements);
+
+    float64_array cir_a = res["fields/circle_a/values"].value();
+    float64_array cir_b = res["fields/circle_b/values"].value();
+    float64_array cir_c = res["fields/circle_c/values"].value();
+
+    float64_array mat_bg = res["matsets/matset/volume_fractions/background"].value();
+    float64_array mat_ca = res["matsets/matset/volume_fractions/circle_a"].value();
+    float64_array mat_cb = res["matsets/matset/volume_fractions/circle_b"].value();
+    float64_array mat_cc = res["matsets/matset/volume_fractions/circle_c"].value();
+    
+    for(index_t idx = 0; idx < elements; idx++)
+    {
+        mat_ca[idx] = cir_a[idx];
+        mat_cb[idx] = cir_b[idx];
+        mat_cc[idx] = cir_c[idx];
+        mat_bg[idx] = 1. - (cir_a[idx] + cir_b[idx] + cir_c[idx]);
+    }
+}
+
+void make_sparse(Node & src, index_t len, Node & n)
+{
+    float64_array src_val = src.value();
+
+    index_t nz = 0;
+    for (index_t idx = 0; idx < len; ++idx)
+    {
+        if (src_val[idx] > 0)
+        {
+            nz += 1;
+        }
+    }
+
+    n["values"].set(DataType::float64(nz));
+    n["element_ids"].set(DataType::int32(nz));
```

Comment:
Looking at how this is called, it looks like it will create the following structure:

```
matsets:
  matset:
    volume_fractions:
      circle_a:
        values: [ ... ]
        element_ids: [ ... ]
      circle_b:
        values: [ ... ]
        element_ids: [ ... ]
      circle_c:
        values: [ ... ]
        element_ids: [ ... ]
```

Based on the latest from #511, I think it should instead be structured like:

```
matsets:
  matset:
    volume_fractions:
      circle_a: [ ... ]
      circle_b: [ ... ]
      circle_c: [ ... ]
    element_ids:
      circle_a: [ ... ]
      circle_b: [ ... ]
      circle_c: [ ... ]
```

The schema described in #512 is currently out of date, so this is largely my fault. I will update it presently so that this isn't unclear going forward.

Reply:
Changed to the structure you wrote here.